tutor/src/screens/my-courses/new-teacher.tsx

```diff
@@ -443,6 +443,10 @@ const SubjectSelect: React.FC<SubjectSelectProps> = ({
     }
 
     const onChangeSuggestion = (value) => {
+        if (value.length > 0) {
```

Comment:
I think you can do `if(value)` since `''` is a falsey value.

Reply:
Ah good tip!